pages/arduino-info.mdx

```diff
@@ -0,0 +1,129 @@
+import Head from 'next/head'
+import Meta from '@hackclub/meta'
+
+<Meta
+  as={Head}
+  name="Summer of Making"
+  title="About your Arduino Starter Kit"
+/>
+
+<main className="container">
```

Comment:
You can also do `<Container as={BaseStyles} />` if you want to use theme-ui here, but whatever you want

Reply:
🤷 I'll try it out

Reply:
hmmm idk what I've done :/